### ISSUE TITLE
Open a link from  a contaier tab  to new no isolated tab for specific domains

### ISSUE THREAD

**Issue by KonstantMan** [stoically/temporary-containers#531]
I can't  find  a way to disable isolation when  clicking a link inside a container 

Excluding domains globally works only with bookmarks and when opening a site from a normal tab 

For example i want to open a github link without an isolation even though the link was in an isolated site

**Comment by crssi** [comment]
Why would you want to do that?
Are there many sites you visit where you need exceptions like that?

Cheers

**Comment by KonstantMan** [comment]
I need that for sites i want to keep some cookies and load fast like brave search and github

**Comment by crssi** [comment]
I see. In addition to TC I am using [Containerise](https://addons.mozilla.org/firefox/addon/containerise/) for exactly the same purpose. 😉 

Then you just add rule for example for github in my case `*.github.com , GITHUB`
I have about a dozen sites in the config and that is it.

Cheers 😄

**Comment by KonstantMan** [comment]
Yeah but i want to find a way to do it with this extension since it's already working as i said  "Excluding domains globally works only with bookmarks and when opening a site from a normal tab"

**Comment by stoically** [comment]
Seems like you misconfigured something, maybe provide your exact configuration.

If I open `example.org` in a new container tab and have `www.iana.org` globally excluded, it stays in the same container for me.

**Comment by KonstantMan** [comment]
You see that's  exactly what i don't want i want specific domains to open in their pre-configured settings even if they are opened from a link inside a container

**Comment by stoically** [comment]
Ah, so you're actually looking for a way to force pages to open in no container. Yeah, that's not supported by TC currently and is different from excluding things from isolation. The only Add-on I'm aware of that supports that is the one mentioned by @crssi, Containerise.

**Comment by KonstantMan** [comment]
At least is it possible to have the same cookies through different containers ?

Because i want the same configuration for brave search  and github no matter if i open them from bookmarks or  in a contained tab

**Comment by crssi** [comment]
I will say this once again: You can achieve that by utilizing Containerise. 😉

**Comment by KonstantMan** [comment]
And  will i say this again i will not use 3 extensions just for isolation 2 should be more than enough , firefox is already slow i will not add extensions just for one thing

**Comment by stoically** [comment]
> At least is it possible to have the same cookies through different containers ?

Not through different containers, no.

> Because i want the same configuration for brave search and github no matter if i open them from bookmarks or in a contained tab

You'd need to pin github and brave to the same permanent container. TC is not about permanent containers generally.
